lk2nd/device/dts/msm8953/msm8953-xiaomi-daisy.dts

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	qcom,msm-id = <QCOM_ID_MSM8953 0>;
+	qcom,board-id = <QCOM_BOARD_ID(MTP, 1, 0) 0x09>;
+};
+
+&lk2nd {
+	model = "Xiaomi Mi A2 Lite";
+	compatible = "xiaomi,daisy";
+	lk2nd,match-bootloader = "*TISSOT*";
```

Comment:
hm is "daisy" presenting itself as "tissot"? the same way as the "tissot" in mi a1?

Reply:
`androidboot.bootloader=MSM8953_DAISY2.0_20190717191738`
From console-ramoops of mi a2 lite so it seems so